Server/src/main/java/com/assignment/concurrent/domain/PointPairingTask.java

```diff
@@ -0,0 +1,77 @@
+package com.assignment.concurrent.domain;
+
+import java.sql.Time;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Random;
+import java.util.Set;
+import java.util.concurrent.Callable;
+
+import com.assignment.concurrent.service.PointsService;
+
+public class PointPairingTask implements Callable<Edge> {
+
+    private static int totalEdgeFormed;
+    private static long startTime;
+    private boolean threadFail;
+    private int failAttempts;
+    private int numberOfPoints;
+    private Point[] points;
+    private List<Edge> edgeArrayList;
+
+    public PointPairingTask(Point[] arr){
+        this.points = arr;
+        this.edgeArrayList = new ArrayList<Edge>();
+    }
+
+    @Override
+    public Edge call() throws Exception {
+        long start = 0;
+        long now = 0;
+        long target = 2;
+
+        while( (now-start) < target ){
+            edgeArrayList.add(formEdges());
+        }
+        
+        return null;
```

Comment:
@LUXIANZE 
1. `while( (now-start) < target )` is an infinite loop.
2. why `return null` ?

this part will be left for @CarlosSia to work on his part is it?

Reply:
1. Yes, the control will be done by @CarlosSia , perhaps I forgot to add a TODO there.
2. Actually callable is to be incorporated with trylock(), but since trylock is used incorrectly here, then I dunno what to do. But after a discussion with @junzai97, he agrees to change on that part, so I will modify later.

ask anything if have confusion.

Reply:
noted